packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -448,7 +448,20 @@ class TypeMatcher<T> {
 /// updating the underlying element (i.e., by calling [Element.update] with the
 /// new widget). Otherwise, the old element is removed from the tree, the new
 /// widget is inflated into an element, and the new element is inserted into the
-/// tree.
+/// tree. Basically any time you're building a list of widgets from a list of
+/// data, and the list might change, you should key the children, so that we
```

Comment:
[key] instead of key ?

Reply:
Here I'm not trying to reference the property but trying to say that one should set a key to the children.